client_plugin/drivers/shared/shared_driver.go

```diff
@@ -62,6 +86,18 @@ func NewVolumeDriver(cfg config.Config, mountDir string) *VolumeDriver {
 	}
 	d.dockerd = cli
 
+	// Pull Samba image
+	log.Infof("Pulling Samba server image")
```

Comment:
Is it possible to download the image when we create the plugin?
And we only need to load it when it's needed.